apps/vx-scan/frontend/src/api.ts

```diff
@@ -14,3 +21,42 @@ export function useApiClient(): grout.Client<Api> {
   }
   return apiClient;
 }
+
+export const queryClientDefaultOptions: DefaultOptions = {
+  queries: {
+    // Since our backend is always local, we don't want react-query to "pause"
+    // when it can't detect a network connection.
+    networkMode: 'always',
+
+    // Never mark cached data as stale automatically. This will prevent
+    // automatic refetching of data (e.g. upon navigating to a page). Cached
+    // queries will only be refecthed when we explicitly invalidate the query
+    // after a mutation. This is an appropriate strategy in VxSuite since
+    // there is only ever one frontend client updating the backend, so we
+    // don't expect data to change on the backend except when we mutate it.
+    staleTime: Infinity,
+  },
+  mutations: { networkMode: 'always' },
+};
```

Comment:
Likely too preemptive for now but could be nice to create a shared config for all the Vx frontend apps so that these settings don't drift from app to app over time - can of course wait till we migrate another app to the new architecture